dspot/src/main/java/eu/stamp_project/dspot/selector/PitMutantScoreSelector.java

```diff
@@ -36,19 +35,22 @@
 
     private int numberOfMutant;
 
-    private List<PitResult> originalKilledMutants;
+    private List<AbstractPitResult> originalKilledMutants;
 
-    private Map<CtMethod, Set<PitResult>> testThatKilledMutants;
+    private Map<CtMethod, Set<AbstractPitResult>> testThatKilledMutants;
 
-    private List<PitResult> mutantNotTestedByOriginal;
+    private List<AbstractPitResult> mutantNotTestedByOriginal;
+
+    private AbstractParser parser;
 
     public PitMutantScoreSelector() {
         this.testThatKilledMutants = new HashMap<>();
+        parser = new PitXMLResultParser();
     }
 
     public PitMutantScoreSelector(String pathToOriginalResultOfPit) {
         this();
-        initOriginalPitResult(PitResultParser.parse(new File(pathToOriginalResultOfPit)));
+        initOriginalPitResult(parser.parse(new File(pathToOriginalResultOfPit)));
```

Comment:
We may have a problem here. In fact, the `pathToOriginalResultOfPit` come from the command line.

I suggest that: we update the command line options [`(m|--path-pit-result)`](https://github.com/STAMP-project/dspot/blob/master/dspot/src/main/java/eu/stamp_project/utils/options/JSAPOptions.java#L198) in order to:
say that we support also the XML format and according to the given format, we use the correct parser.

Today, one may want to you the CSV format for the initial mutation score but want to use the XML as current output.